caddyhttp/rewrite/rewrite_test.go

```diff
@@ -131,6 +131,45 @@ func TestRewrite(t *testing.T) {
 	}
 }
 
+// TestWordpress is a test for wordpress usecase.
+func TestWordpress(t *testing.T) {
```

Comment:
Does this need to be Wordpress specific?

Reply:
not really, I was using to test while writing the code and decided to leave it there and let you all judge if it should stay.

Reply:
Okay, I'll leave this to Matt :smile:

Reply:
I think its fine to leave this as a test.